public/index.html

```diff
@@ -253,6 +256,8 @@ <h5 class="widgetSubTitle" style="margin-bottom: 20px;">Customization Options (O
 						<option value="pt-br">PT-BR</option>
 					</select>
 					<h5 class="widgetSubTitle" style="margin-bottom: 20px;">Customization Options (Optional): </h2>
+						<input type="text" class="form-control" onblur="changeAttribute('treeProfile',this.value,'forestname')"
+							placeholder="Forest Name"><br />
 						<div class="form-check form-switch">
```

Comment:
Move the switch below the them selector - currently separating Forest name and theme (compare design of Tree-Map Widget):

<img width="302" alt="Bildschirmfoto 2021-06-03 um 12 56 39" src="https://user-images.githubusercontent.com/1532418/120634265-56b24300-c46b-11eb-8b77-590012799527.png">

Reply:
fixed that myself